config/locales/bulk_export/bulk_export-fr.yml

```diff
@@ -0,0 +1,53 @@
+      show_details: Afficher les détails
+      status: Statut
+      upload_details: Télécharger les détails
+      user: Utilisateur
+    new:
+      download: Télécharger
+      expanded_plaintext: Texte en clair étendu
+      expanded_plaintext_description: Comme le texte brut verbatim, ce fichier de texte brut représentera les sauts de ligne avec un seul saut de ligne, les sauts de paragraphe avec un double saut de ligne et les sauts de page avec un triple saut de ligne. Il diffère du texte verbatim, en ce que la normalisation sera appliquée à tous les sujets mentionnés, de sorte que si le texte verbatim peut se lire "J'ai salué M. Jones et sa femme ce matin.", le texte en clair modifié se lira "J'ai salué James Jones et Elizabeth Smith Jones ce matin ». Ce texte artificiel est utile pour l'analyse programmatique, mais n'est pas destiné à être lu par des humains.
+      export_all_works: Exporter toutes les œuvres
+      facing_edition_pdf: Face à l'édition PDF
+      facing_edition_pdf_description: Un fichier PDF contenant des images et des transcriptions sur les pages opposées.
+      html: HTML
+      html_description: Cela peut être utile pour la conservation dans d'autres systèmes ou comme point de départ pour l'affichage sur un autre site Web.
+      one_file_per_collection: Un fichier par collection
+      one_file_per_page: Un fichier par page
+      one_file_per_work: Un fichier par œuvre
+      one_site_per_collection: Un site par collection
+      previous_exports: Exportations précédentes
+      search_optimized_plaintext: Texte brut optimisé pour la recherche
+      search_optimized_plaintext_description: Une version en clair de l'œuvre optimisée pour la recherche en texte intégral. Cette version contient une transcription textuelle en clair de chaque page (comme décrit ci-dessus), sauf que les mots coupés par des sauts de ligne sont joints ensemble, et une liste des noms canoniques mentionnés dans chaque page est ajoutée à la fin de la page.
+      start_export: Démarrer l'exportation
+      static_site: Site statique
+      static_site_description: Un site Jekyll statique contenant l'intégralité de l'édition
+      status: Statut
+      subject_csv: Objet CSV
+      subject_csv_description: Une feuille de calcul répertoriant chaque endroit où un sujet est mentionné dans une page de la collection
+      table_field_csv: Tableau/Champ CSV
+      table_field_csv_description: Exporte une feuille de calcul avec des données basées sur des champs ou tabulaires.
+      tei_xml: XML TEI
+      tei_xml_description: Cela peut être utile aux éditeurs qui envisagent de faire un balisage supplémentaire dans les éditeurs TEI-XML comme oXygen.
+      text_docx: Texte DOCX
+      text_docx_description: Un fichier MS-Word (.docx) contenant des transcriptions de texte.
+      text_pdf: Texte PDF
+      text_pdf_description: Un fichier PDF contenant des transcriptions de texte.
+      time: Temps
+      verbatim_plaintext: Texte brut textuel
```

Comment:
I really don't like this, which I'd translate to "textual raw text".  @saracarl what do you think we should do about bad translations at this point?